src/katsdpcontroller/generator.py

```diff
@@ -762,6 +762,9 @@ def get_baseline_index(a1, a2):
         Sensor(int, f"{stream.name}-n-chans-per-substream",
                "Number of channels in each substream for this x-engine stream",
                default=stream.n_chans_per_substream, initial_status=Sensor.Status.NOMINAL),
+        Sensor(float, f"{stream.name}.ddc-mix-freq",
+               "F-engine DDC mixer frequency, where used. 0 where n/a",
+               default=0, initial_status=Sensor.Status.NOMINAL),
```

Comment:
Might request you add the `units` kwarg, like in `fake_sever.py`'s `dig-pwr-dbfs` (or katgpucbf's `dsim/server.py` for `adc-sample-rate`).

Reply:
That's a good point.

Reply:
Out of interest, any reason we went with `mix-freq` instead of `mixer-freq`?

Reply:
Uh, now you're asking the hard questions. I just went with what's written in the ICD, I think it's what was used previously.

Reply:
The original ICD seemed to name things like they were being charged by the letter. So bls, chans, ants, mix, pwr instead of baselines, channels, antennas, mixer, power.